[2024-03-18T06:41:11.036+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: two_separate_groups.bigquery_external_table_4_task manual__2024-03-18T06:40:31.356042+00:00 [queued]>
[2024-03-18T06:41:11.214+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: two_separate_groups.bigquery_external_table_4_task manual__2024-03-18T06:40:31.356042+00:00 [queued]>
[2024-03-18T06:41:11.226+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-18T06:41:11.458+0000] {taskinstance.py:2214} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_4_task> on 2024-03-18 06:40:31.356042+00:00
[2024-03-18T06:41:11.549+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'two_separate_groups', 'bigquery_external_table_4_task', 'manual__2024-03-18T06:40:31.356042+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/ingest_data_to_gcs.py', '--cfg-path', '/tmp/tmpluwqb2ag']
[2024-03-18T06:41:11.484+0000] {standard_task_runner.py:60} INFO - Started process 252 to run task
[2024-03-18T06:41:11.566+0000] {standard_task_runner.py:88} INFO - Job 212: Subtask bigquery_external_table_4_task
[2024-03-18T06:41:12.194+0000] {task_command.py:423} INFO - Running <TaskInstance: two_separate_groups.bigquery_external_table_4_task manual__2024-03-18T06:40:31.356042+00:00 [running]> on host 9958f290dc4a
[2024-03-18T06:41:13.033+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='two_separate_groups' AIRFLOW_CTX_TASK_ID='bigquery_external_table_4_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T06:40:31.356042+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T06:40:31.356042+00:00'
[2024-03-18T06:41:13.063+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-03-18T06:41:13.068+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-18T06:41:13.089+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-18T06:41:15.181+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=two_separate_groups, task_id=bigquery_external_table_4_task, execution_date=20240318T064031, start_date=20240318T064111, end_date=20240318T064115
[2024-03-18T06:41:15.438+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T06:41:15.670+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
